src/foundry/diceTerms/mathTerm.d.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * A type of RollTerm used to apply a function from the Math library.
+ */
+declare class MathTerm extends RollTerm {
+  /**
+   * @param termData - Data used to create the MathTerm, including the following:
+   *                   (default: `{}`)
+   * @param fn - The named function in the Math environment which should be applied to the term
+   * @param terms - An array of string argument terms for the function
+   */
+  constructor(termData?: Partial<MathTerm.TermData>);
```

Comment:
```suggestion
  constructor({ termData, terms, options }?: Partial<MathTerm.TermData>);
```

Reply:
That doesn't match the signature of the constructor, but I get your point